packages/genesys-spark-components/src/components/stable/gux-form-field/helper-components/gux-form-field-tooltip/gux-form-field-tooltip.tsx

```diff
@@ -0,0 +1,39 @@
+import { Component, Element, h, JSX, Prop } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxFormFieldTooltipVariant } from './gux-form-field-tooltip.types';
+
+@Component({
+  styleUrl: 'gux-form-field-tooltip.scss',
+  tag: 'gux-form-field-tooltip'
+})
+export class GuxFormFieldTooltip {
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  variant: GuxFormFieldTooltipVariant = 'info';
+
+  @Prop()
+  screenreaderText: string;
+
+  componentWillLoad(): void {
+    trackComponent(this.root);
+  }
+
+  render(): JSX.Element {
+    return (
+      <div class="gux-form-field-tooltip">
+        <gux-icon
+          id="form-field-tooltip"
```

Comment:
Do we need to generate a random ID or is this ok as its inside the shadow dom?

Reply:
I added a random ID just to be on the safe side.